README.md

```diff
@@ -235,6 +236,7 @@ ros2 run gazebo_ros2_control_demos example_position
 ros2 run gazebo_ros2_control_demos example_velocity
 ros2 run gazebo_ros2_control_demos example_effort
 ros2 run ign_ros2_control_demos example_diff_drive
```

Comment:
Is ign_ros2_control_demos intentional here?

Reply:
nope, it's wrong too.

Do you have any plans to include this demo in `ign_ros2_control` ?

Reply:
No not at the moment, maybe when I migrate to ignition